examples/99_hello_sphere/ex_hello_sphere/main.rb

```diff
@@ -28,7 +28,7 @@ def self.create_sphere
 
     unless file_loaded?(__FILE__)
       menu = UI.menu('Plugins')
-      menu.add_item('Create Sphere Example') {
+      menu.add_item('99 Create Sphere Example') {
```

Comment:
The 99 prefix was something I did while originally working on this. At the time I didn't know where they'd fit into the progression of the examples. Maybe worth to revisit.

Reply:
I've made a list of possible things to clean up. Adding this there too.